Source/Core/VideoCommon/VertexLoaderManager.h

```diff
@@ -24,6 +24,10 @@ namespace VertexLoaderManager
 	void AppendListToString(std::string *dest);
 
 	NativeVertexFormat* GetCurrentVertexFormat();
+
+	// Position cache for zfreeze (+ 4 bytes to compensate for SIMD overwrite)
+	extern float position_cache[3 * 3 + 4];
```

Comment:
Heh, I just changed that today. Maybe I should trust myself more when I'm actually writing code.